editions/tw5.com/tiddlers/filters/syntax/All Filter Run Prefix.tid

```diff
@@ -0,0 +1,17 @@
+created: 20230316150731234
+from-version: 5.1.23
+modified: 20230319170500368
+rp-input: all titles from previous filter runs
+rp-output: output titles are appended to the filter's output without de-duplication. <<.from-version "5.1.20">>
+rp-purpose: union of sets without de-duplication
+tags: [[Named Filter Run Prefix]]
+title: All Filter Run Prefix
+type: text/vnd.tiddlywiki
+
+<$railroad text="""
+\start none
+( ":all" | - )
+[[run|"Filter Run"]]
```

Comment:
I was experimenting a lot. Having the link box there does make the navigation much more logical and it also shows the context of the prefix. I think if there is the prefix alone we can remove the diagram completely. 

I was thinking to add 2 more possibilities to the railroad-plugin. .. 1st. A label at the beginning, which would allow a link back to the "parent" diagram. 2nd a possibility to get "dotted border" for the link box. ... or any other styling that would make it clearer that it is there for the link only

Reply:
@pmario: I think showing the current item in context is not a bad idea, when it is clear that other parts are just the surrounding context. Maybe semi-transparency could work, too?